test/strategies.spec.ts

```diff
@@ -0,0 +1,177 @@
+import { afterEach, describe, it } from "mocha";
+import * as sinon from "sinon";
+import * as CurrentContext from "../src/current_context";
+import { expect } from "chai";
+import { IterateableContext, isToggleEnabled } from "../src/strategies";
+import { Toggle } from "../src/client";
+import { ContextData } from "../lib/current_context";
```

Comment:
error here

Reply:
Please describe error

Reply:
cant find the path. (pat is not correct)

Reply:
updated

Reply:
not yet :)